contracts/package.json

```diff
@@ -17,7 +17,8 @@
     "@openzeppelin/contracts": "^4.3.2",
     "dotenv": "^10.0.0",
     "ethers": "^5.5.1",
-    "hardhat": "^2.6.7"
+    "hardhat": "^2.6.7",
+    "web3": "^1.7.0"
```

Comment:
This (and a number of other dependencies) could be a `devDependencies` as it/they are not needed for contracts directly. @jzaki Any thoughts on separation of prod/dev dependencies for the contracts project? It's more relevant for projects where an npm package/runnable binary is produced and we want to keep it slim, but might be good for separation here as well.

Reply:
Good point. In the context of contracts, @openzeppelin/contracts is a dependency, and most others are `devDependencies`

Reply:
:+1: Moved to https://github.com/jzaki/bls-wallet/issues/139